src/releng_docs/deploying.rst

```diff
@@ -1,146 +1,244 @@
 Deploying
 =========
 
-Merging to either ``staging`` or ``production`` branches will, if build is
-successfull, also trigger a deployment script. Deployment script will only
-deploy services that were changed, since last deployment.
+Deployment happens **automatically** in :ref:`continuous integration
+<continuous-integration>` when code is pushed / merged to **staging** or
+**production** branch.
 
-``mozilla-releng/services`` build system is flexible enough to deploy to many
-different targets. Currently we deploy services to Heroku and Amazon S3, but
-this can be extended to any other target.
+If interested please read more on :ref:`branching policy <branching-policy>`.
 
-Deploying of new services - at this time - does not handle creation/removal of
-S3 buckets / Heroku applications. Those need to be created beforehand.
 
-Manually deploying
-------------------
+Release Schedule
+----------------
 
-To deploy a single service manually, you need to run:
+Releases happen on a **weekly** basis, excluding `Firefox release week`_. `Next
```

Comment:
Do you want to mention a specific day here? Wednesday is generally the safest day (no releases building or shipping). You may also want to note that releases may be skipped due to chemspills.